spec/receipt_printer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'receipt_printer'
+
+describe ReceiptPrinter do
+  subject(:receipt_printer) {described_class.new(restaurant_data, totals)}
+  let(:restaurant_data) {{:shopname => :the_coffee_connection,
+                          :address => :'123_lakeside_way',
+                          :phone => "16503600708"}}
+  let(:totals) { {subtotal: {cafe_latte: "2 x $9.50", blueberry_muffin: "1 x $4.05"}, tax: "$1.17", total: "$14.72"} }
```

Comment:
Metrics/LineLength: Line is too long. [118/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)